test/validation/api/event/event.c

```diff
@@ -18,7 +18,7 @@ static void event_test_free(void)
 	odp_buffer_t buf;
 	odp_packet_t pkt;
 	odp_timeout_t tmo;
-	odp_event_subtype_t subtype;
+	odp_event_subtype_t subtype = ODP_EVENT_NO_SUBTYPE;
```

Comment:
As before, `ODP_EVENT_PACKET_BASIC` could be used here.

Reply:
Fixed in V2